source/openpulse/openpulse/parser.py

```diff
@@ -310,14 +331,17 @@ def visitOpenpulseStatement(self, ctx: openpulseParser.OpenpulseStatementContext
 class CalParser(QASMVisitor[None]):
     """Visit OpenQASM3 AST and pase calibration"""
```

Comment:
```suggestion
    """Visit OpenQASM3 AST and parse calibration."""
```
Should we also document the `permissive` argument here? Also happy to just say that this class is an internal detail.

Reply:
Not a bad idea -- I updated the docstring.